data/xp-pen-artist10s.tablet

```diff
@@ -0,0 +1,28 @@
+# XP-Pen
+# Artist 10S (1st generation)
+#
+
+[Device]
+Name=UGEE 10.1 Tablet Monitor
+ModelName=XP-Pen Artist 10S (1st gen)
```

Comment:
```suggestion
ModelName=
```
ModelName doesn't really apply to most XP pen/UGEE tablets, more to Wacom where the "Wacom Intuos blah" has a model name of "ABC-123".

Reply:
fixed